controllers/openstackclient_controller.go

```diff
@@ -139,7 +142,7 @@ func (r *OpenStackClientReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	}(cond)
 
 	// If we determine that a backup is overriding this reconcile, requeue after a longer delay
-	overrideReconcile, err := ospdirectorv1beta1.OpenStackBackupOverridesReconcile(r.Client, instance.Namespace, true)
+	overrideReconcile, err := ospdirectorv1beta1.OpenStackBackupOverridesReconcile(r.Client, instance.Namespace, instance.Status.Conditions.GetCurrentCondition() != nil && instance.Status.Conditions.GetCurrentCondition().Reason == ospdirectorv1beta1.OsClientCondReasonPodProvisioned)
```

Comment:
with the above `cond := instance.Status.Conditions.GetCurrentCondition()` shouldn't it be ok to just check cond.Reason? it should never be nil here, right?

Reply:
Yep, and that will be cleaner to read as well

Reply:
Done